index.bs

```diff
@@ -130,8 +147,12 @@ or a combination of both.
 
 This specification relies on several other underlying specifications.
 
-: HTML5
-:: The concept of <dfn for='web'>origin</dfn> and the <dfn>Navigator</dfn> interface are defined in [[!HTML5]].
+<!-- , <a>opaque origin</a>, -->
+
+: HTML
+:: The concepts of <a link-for='web'>current settings object</a>, |given value|, <a link-for='web'>opaque origin</a>, 
```

Comment:
hm, well, given this branch's current state, it needs the link-for='web' to compile.  perhaps we can clean that up after merging all these PRs to master?